assemblies/assembly_filtering-process-instances/main.adoc

```diff
@@ -0,0 +1,28 @@
+[id='assembly_filtering-process-instances']
```

Comment:
The sections are not shown in this PR, but I would also update the following:
- Change "Overview" to "Business processes in {CENTRAL}", and I'd tweak the first sentence to "A business process application created by a citizen developer in Business Central depicts ..."
- "Knowledge work tasks in {CENTRAL}"

Reply:
Changed as suggested.